src/shared/components/Tooltip/Tooltip.style.ts

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import { colors, sizes, typography, zIndex, transitions } from '@/shared/theme'
+
+type TooltipProps = {
+  isActive?: boolean
+  arrowDisabled?: boolean
+}
+
+export const StyledTooltip = styled.div<TooltipProps>`
+  position: relative;
+  width: fit-content;
```

Comment:
Do we need that? I see the support for `fit-content` isn't great, although we could use it. Could give me some context why we need that?

Reply:
Tooltip takes whole horizontal space, it does not take children's width into account. I wanted avoid that. Alternatively I could use `display: inline-block` instead